features/datachoices/src/main/java/org/opennms/features/datachoices/internal/UsageStatisticsReporter.java

```diff
@@ -175,12 +187,27 @@ public UsageStatisticsReportDTO generateReport() {
         usageStatisticsReport.setNodes(m_nodeDao.countAll());
         usageStatisticsReport.setIpInterfaces(m_ipInterfaceDao.countAll());
         usageStatisticsReport.setSnmpInterfaces(m_snmpInterfaceDao.countAll());
+        usageStatisticsReport.setSnmpInterfacesWithFlows(m_snmpInterfaceDao.getNumInterfacesWithFlows());
         usageStatisticsReport.setMonitoredServices(m_monitoredServiceDao.countAll());
         usageStatisticsReport.setEvents(m_eventDao.countAll());
         usageStatisticsReport.setAlarms(m_alarmDao.countAll());
+        usageStatisticsReport.setSituations(m_alarmDao.getNumSituations());
+        usageStatisticsReport.setMonitoringLocations(m_monitoringLocationDao.countAll());
+        usageStatisticsReport.setMinions(m_monitoringSystemDao.getNumMonitoringSystems(OnmsMonitoringSystem.TYPE_MINION));
+        usageStatisticsReport.setSentinels(m_monitoringSystemDao.getNumMonitoringSystems(OnmsMonitoringSystem.TYPE_SENTINEL));
```

Comment:
I think we don't add any sentinels to Monitoring systems. Should we postpone counting sentinels till we address that ?

Reply:
Right. If they are not added, then we should omit.

Reply:
Removed.